en-US/firefox/new/horizon.lang

```diff
@@ -104,3 +118,28 @@ Freedom is yours
 
 ;Use the only browser built for people, not profit.
 Use the only browser built for people, not profit.
+
+
+## TAG: firefox_new_other_platforms
+;Firefox for other platforms
+Firefox for other platforms
+
+
+## TAG: firefox_new_other_platforms
+;Download Firefox <span>for Windows</span>
+Download Firefox <span>for Windows</span>
+
+
+## TAG: macos_branding
+;Download Firefox <span>for macOS</span>
+Download Firefox <span>for macOS</span>
```

Comment:
Are you saying since the purpose of the tag for macos_branding is to update OS X to macOS, from l10n perspective, the actual string to be localized at this point is the same between the two tagged strings, which is "Download Firefox <span>for macOS</span>".   Tag firefox_new_other_platforms will take care of the update.